### ISSUE TITLE
Client reconnects every 60secs with redis-sentinel

### ISSUE THREAD

**Issue by c1aphas** [luin/ioredis#694]
Hello.
I wrote simple demo test.js file:
```
var Redis = require('ioredis');

var ri = new Redis({
  name: 'redis',
  sentinels: [
    {host: 'redis01...', port: 6380},
    {host: 'redis02...', port: 6380},
    {host: 'redis03...', port: 6380},
    {host: 'redis04...', port: 6380},
  ],
});

ri.on('ready', () => console.log('Redis ready'));
ri.on('connect', () => console.log('Redis connected'));
ri.on('close', () => console.log('Redis connection closed'));
ri.on('reconnecting', () => console.log('Redis reconnecting'));
ri.on('error', (e) => console.error(e));
```

So, i see every 60 seconds (w/o errors):
```
Redis connection closed
Redis reconnecting
Redis connected
Redis ready
```
Is that an expected behaviour? Can you explain it please?
nodejs version: 8.11.4

**Comment by luin** [comment]
Could you enable the debug mode `DEBUG=ioredis:* node yourapp.js` and post the logs here?

**Comment by c1aphas** [comment]
Thank you for a quick response.
Here is the log:
```
  ioredis:redis status[localhost:6379]: [empty] -> connecting +0ms
  ioredis:redis status[redis01.head.rambler.tech:6380]: [empty] -> connecting +4ms
  ioredis:redis queue command[0] -> sentinel(get-master-addr-by-name,redis) +2ms
  ioredis:redis status[10.16.36.27:6380]: connecting -> connect +7ms
  ioredis:redis status[10.16.36.27:6380]: connect -> ready +0ms
  ioredis:connection send 1 commands in offline queue +0ms
  ioredis:redis write command[0] -> sentinel(get-master-addr-by-name,redis) +1ms
  ioredis:redis write command[0] -> sentinel(sentinels,redis) +4ms
  ioredis:SentinelConnector adding sentinel 10.16.36.28:6380 +0ms
  ioredis:SentinelConnector adding sentinel 10.41.36.27:6380 +0ms
  ioredis:SentinelConnector adding sentinel 10.41.36.28:6380 +0ms
  ioredis:SentinelConnector Updated internal sentinels: {"sentinels":[{"host":"redis01.head.rambler.tech","port":6380},{"host":"redis02.head.rambler.tech","port":6380},{"host":"redis03.head.rambler.tech","port":6380},{"host":"redis04.head.rambler.tech","po ... <REDACTED full-length="327"> +2ms
  ioredis:SentinelConnector resolved: 10.16.36.27:6379 +1ms
  ioredis:redis status[10.16.36.27:6379]: connecting -> connect +5ms
  ioredis:redis write command[0] -> info() +0ms
Redis connected
  ioredis:redis status[10.16.36.27:6379]: connect -> ready +2ms
Redis ready
  ioredis:redis status[10.16.36.27:6380]: ready -> close +1ms
  ioredis:connection skip reconnecting since the connection is manually closed. +13ms
  ioredis:redis status[10.16.36.27:6380]: close -> end +1ms

  >>>HERE IS 60 SECS IDLE<<<

  ioredis:redis status[10.16.36.27:6379]: ready -> close +1m
  ioredis:connection reconnect in 50ms +1m
  ioredis:redis status[10.16.36.27:6379]: close -> reconnecting +1ms
Redis connection closed
Redis reconnecting
  ioredis:redis status[10.16.36.27:6379]: reconnecting -> connecting +51ms
  ioredis:redis status[redis01.head.rambler.tech:6380]: [empty] -> connecting +0ms
  ioredis:redis queue command[0] -> sentinel(get-master-addr-by-name,redis) +1ms
  ioredis:redis status[10.16.36.27:6380]: connecting -> connect +1ms
  ioredis:redis status[10.16.36.27:6380]: connect -> ready +1ms
  ioredis:connection send 1 commands in offline queue +55ms
  ioredis:redis write command[0] -> sentinel(get-master-addr-by-name,redis) +0ms
  ioredis:redis write command[0] -> sentinel(sentinels,redis) +0ms
  ioredis:SentinelConnector Updated internal sentinels: {"sentinels":[{"host":"redis01.head.rambler.tech","port":6380},{"host":"redis02.head.rambler.tech","port":6380},{"host":"redis03.head.rambler.tech","port":6380},{"host":"redis04.head.rambler.tech","po ... <REDACTED full-length="327"> +1m
  ioredis:SentinelConnector resolved: 10.16.36.27:6379 +1ms
  ioredis:redis status[10.16.36.27:6379]: connecting -> connect +2ms
  ioredis:redis write command[0] -> info() +1ms
Redis connected
  ioredis:redis status[10.16.36.27:6380]: ready -> close +0ms
  ioredis:connection skip reconnecting since the connection is manually closed. +3ms
  ioredis:redis status[10.16.36.27:6380]: close -> end +0ms
  ioredis:redis status[10.16.36.27:6379]: connect -> ready +0ms
Redis ready
```

**Comment by c1aphas** [comment]
Thank you! I'm closing this.
